platform/ios/PolyPodApp/PodApi/Network/Network.swift

```diff
@@ -39,7 +39,14 @@ final class Network: NetworkProtocol {
         body: String?,
         contentType: String?,
         authToken: String?) -> Result<Data, PodApiError>  {
-            var request = URLRequest(url: URL(string: url)!)
+            let requestURL = URL(string: url)!
```

Comment:
I was wondering what the case would be as using a random String still worked and we could not get it to break to pass an error from there. (I tried guard with else and .failure in the guard, but Strings like "ksjebfwkhe" worked to craft the url, which I thought was weird

Reply:
Not quite weird, URL is also used to access files in FileSystem, so it may be referring to a file in current active directory. 

Something like this would fail `URL(string: "\\someting.com")`

Reply:
We perceived it as weird because it was different in terms of behaviour in kotlin and js where it would throw. But yeah makes sense what you're saying